### PR TITLE
Jayden cesar validate info

### PR DESCRIPTION
**Pull request by Jay7vn**

### CODE REVIEW

**Comment by christopherjlo** [comment]
Please DM me on discord with the contents of your .env file since it's not pushed in the repo (which is correct).
Also, currently database.js has code that looks like a merge didn't finish in the web editor (starting with <<<< HEAD), please fix and commit again.

File: server/config/database.js
Comment (christopherjlo) [inline server/config/database.js:1]: Pls fix line 5 - 15

**Review by christopherjlo — CHANGES_REQUESTED** [review]
see comment on database.js

**Review by christopherjlo — APPROVED** [review]

**Comment by christopherjlo** [comment]
# DO NOT MERGE THIS PR YET

@chillwafflez @DanielPasion we need to discuss and standardize controllers and routes before merging anything cuz otherwise merging will get really messy

**Comment by DanielPasion** [comment]
> # DO NOT MERGE THIS PR YET
> @chillwafflez @DanielPasion we need to discuss and standardize controllers and routes before merging anything cuz otherwise merging will get really messy



> Please DM me on discord with the contents of your .env file since it's not pushed in the repo (which is correct). Also, currently database.js has code that looks like a merge didn't finish in the web editor (starting with <<<< HEAD), please fix and commit again.

Caleb specified what the .env files were already for the env file its the aws setup stuff.

**Comment by MichaelWuhu** [comment]
I added password hashing (for create acc route) and moved some stuff around to fit what the tech leads chose to standardize. 

Make sure that everythin works as it shuld (i used the name 'userLogin' for most things related to this task) and then delete any old/unused files/commets/folders then re-ping a tech lead to see if we can merge this into main

don't forget to run 'npm install' before u start testing
(also i did have to change up ur guys' package.json so you might need to delete all node modules then rerun 'npm install')

@Jay7vn @cesar-henry

**Pull request closed without merging**
